virtManager/delete.py

```diff
@@ -493,3 +491,25 @@ def append_str(str1, str2, delim="\n"):
         log.exception("Failed checking disk conflict: %s", str(e))
 
     return (not info, info)
+
+
+class vmmDeleteDialog(_vmmDeleteBase):
+
+    def __init__(self):
+        _vmmDeleteBase.__init__(self)
+
+    def _get_dialog_title(self):
+        return self.vm.get_name()
+
+
+class vmmDeleteStorage(_vmmDeleteBase):
+
+    disk = None
+    def __init__(self):
+        _vmmDeleteBase.__init__(self)
+
```

Comment:
I think you can change the __init__ function here to be

```python
def __init__(self, disk):
    _vmmDeleteBase.__init__(self)
    self.disk = disk
```

Rather than have a separate `set_disk` function